org.vergecurrency.verge-qt.json

```diff
@@ -0,0 +1,69 @@
+{
+  "id": "org.vergecurrency.verge-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "6.0.2",
+  "command": "verge-qt",
+  "finish-args": [
+    /* Wayland and Xorg support */
+    "--socket=x11",
+    "--socket=wayland",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "verge",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/verged",
+        "install -m 0744 -D -t /app/bin/internal bin/verge-qt",
+        "install -m 0744 -D -t /app/bin/internal bin/verge-cli",
+        "install -m 0744 -D -t /app/bin/internal bin/test_verge"
+      ],
+      "post-install": ["/app/bin/internal/test_verge"],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
```

Comment:
if you don't want to build it from source, create `flathub.json` with the same key to limit the build to x86_64.

Reply:
ah good suggestion, i didn't see this in my references. thanks!